packages/vaadin-lumo-styles/components.css

```diff
@@ -1,4 +1,6 @@
 @import './components/button.css';
+@import './components/details-summary.css';
+@import './components/details.css';
```

Comment:
I think these should go together. You won't use one without the other, so I’d combine both into the `details.css` file.

Reply:
Or, well, I suppose someone might just use `<vaadin-details>` with a custom summary element, and might want to avoid the extra CSS.